index.html

```diff
@@ -1,13 +1,57 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta content="IE=edge">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <title>Microverse Portfolio</title>
+        <title>Microverse Adebsa Portfolio</title>
         <link rel="stylesheet" href="style.css">
     </head>
 
     <body>
-        
+        <!-- header start -->
+        <header>
+            <label id="author-name">
+                Adebsa
+            </label>
+            <svg class="icon" id="icon-menu" xmlns="http://www.w3.org/2000/svg" fill="#FFFFFF" class="bi bi-list" viewBox="0 0 16 16">
+                <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+            </svg>
+        </header>
+        <!-- header end -->
+
+        <!-- headline start -->
+        <div id="headline">
+            <h1>Hey There. I'm Ben Salès</h1>
+            <h2>I am a Software Developer</h2>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+
+            <!-- social medias start -->
+            <div id="social-medias">
```

Comment:
- [ ] Kindly use the `section` tag instead of the `div` here.

Reply:
Good job following the advice of the previous reviewer